pom.xml

```diff
@@ -8,7 +8,7 @@
     <version>1.0-SNAPSHOT</version>
     <name>expenses-manager-parent</name>
 
-    <packaging>pom</packaging>
```

Comment:
Our project is a "Multi-Module Project", that means the parent pom packaging should be pom, not jar. 
![image](https://github.com/user-attachments/assets/970cea0f-cd86-4e8c-a210-09c8e9ac7cfa)

You can find more details here https://www.baeldung.com/maven-multi-module